package.json

```
@@ -54,6 +54,7 @@
     "@types/brotli-webpack-plugin": "^1.1.0",
     "app-root-path": "3.0.0",
     "babel-loader": "^8.1.0",
+    "babel-plugin-lodash": "3.3.4",
```

```suggestion
    "app-root-path": "^3.0.0",
    "babel-loader": "^8.1.0",
    "babel-plugin-lodash": "^3.3.4",
```
I know I've said the opposite quite some time, but for `libraries`, we actually want "compatible with" versions so that we avoid duplication of dependencies if we require exact versions (e.g. 3.3.4 from here and maybe 3.3.2 from somewhere else, we'll wind up with 2 versions). If we use `compatible with`, it will just use 3.3.4 for both of them.

for our products, we want exact versions though because we have package-lock anyways and it makes it easier to see what is really installed.